pkg/authentication/handlers.go

```diff
@@ -53,26 +76,57 @@ type API struct {
 	apiKey           string
 	payloadValidator validation.PayloadValidatorInterface
 	oauth2           OAuth2ContextInterface
+	helper           OAuth2HelperInterface
+	authCookiesTTL   time.Duration
 
-	tracer   trace.Tracer
-	logger   logging.LoggerInterface
-	nonceTTL time.Duration
+	tracer        trace.Tracer
+	logger        logging.LoggerInterface
+	cookieManager AuthCookieManagerInterface
```

Comment:
```suggestion
	cookieManager AuthCookieManagerInterface
	
	tracer        trace.Tracer
	logger        logging.LoggerInterface
```

just a matter of standardization, o11y stuff is always in the bottom